README.md

```diff
@@ -17,7 +17,7 @@ images.getAll((err, images) => {
       [
         {
           kind: 'compute#image',
-          selfLink: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/centos-6-v20150710',
+          selfLink: 'https://compute.googleapis.com/compute/v1/projects/centos-cloud/global/images/centos-6-v20150710',
```

Comment:
Does it really still have compute after the .com/?

Reply:
It does 🙈